.github/workflows/browser-tests.yml

```diff
@@ -13,7 +17,7 @@ jobs:
 
         # The job will only run if the pull request is from the same repository.
         # Benchmarks can't run on PRs from forked repos due to comment posting restrictions without a GitHub token.
-        if: ${{ github.event.pull_request == null || github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
The only trigger for this action is the pull_request event so that value will never be null.